app/views/home/index.html.erb

```diff
@@ -1 +1 @@
-<%= render partial: "signup_form" %>
+<div class="map"></div>
```

Comment:
I pulled master into mine recently. I think we're removing this signup form from the home page so I'm going to remove the failing test as well. I think the problem is that the signup form NEEDS a @company in the form_for but there isn't a @company variable in the home controller and so the signup form fails on the home page. I don't think that render partial should have been in there in the home.index.html. Even though when it pushed up tests were passing, that partial was failing in development. My opinion...Not sure if that's correct. But anyway, I removed the failing test.